Source/Core/Core/PowerPC/Jit64/Jit.cpp

```diff
@@ -236,7 +236,7 @@ void Jit64::WriteCallInterpreter(UGeckoInstruction inst)
 {
 	gpr.Flush();
 	fpr.Flush();
-	if (js.isLastInstruction)
+	if (js.op->opinfo->flags & FL_ENDBLOCK)
```

Comment:
I think so, we  should hit https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/Core/PowerPC/Jit64/Jit.cpp#L859
Else all non-fallbacked instructions would also break.